src/file-1.xoo

```diff
@@ -0,0 +1,17 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
+// dummy line 4
+// dummy line 5
+// dummy line 6
+// dummy line 7
+// dummy line 8
+// dummy line 9
+// dummy line 10
+// dummy line 11
+// dummy line 12
+// dummy line 13
+// dummy line 14
+// dummy line 15
+// dummy line 16
+// violate HasTag rule: xoo on line 17
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:41889/organizations/org0/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:41889/project/issues?id=sample&pullRequest=6305&issues=AWkLjYqbCOhZW8HdPe1W&open=AWkLjYqbCOhZW8HdPe1W)

[ISSUE_KEY]: AWkLjYqbCOhZW8HdPe1W